src/main/java/world/bentobox/aoneblock/AOneBlock.java

```diff
@@ -92,22 +98,44 @@ public void onEnable(){
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "visited_island_phase", phManager::getPhaseByLocation);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "visited_island_count", phManager::getCountByLocation);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "my_island_phase", phManager::getPhase);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "my_island_count", phManager::getCount);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "visited_island_next_phase", phManager::getNextPhaseByLocation);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "my_island_next_phase", phManager::getNextPhase);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "my_island_blocks_to_next_phase", phManager::getNextPhaseBlocks);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "visited_island_blocks_to_next_phase", phManager::getNextPhaseBlocksByLocation);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "my_island_percent_done", phManager::getPercentDone);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "visited_island_percent_done", phManager::getPercentDoneByLocation);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "my_island_done_scale", phManager::getDoneScale);
+        getPlugin().getPlaceholdersManager().registerPlaceholder(this, "visited_island_done_scale", phManager::getDoneScaleByLocation);
 
         // Register request handlers
         registerRequestHandler(new IslandStatsHandler(this));
         registerRequestHandler(new LocationStatsHandler(this));
+
+        // Decide if HolographicDisplays is Useable
+        useHolographicDisplays = Bukkit.getPluginManager().isPluginEnabled("HolographicDisplays");
+
+        // Replace Missing Holograms
+        Bukkit.getScheduler().runTaskLater(BentoBox.getInstance(), new Runnable() {
+            @Override
+            public void run() {
+                if (useHolographicDisplays()) {
+                    for (Island island : getIslands().getIslands()) {
+                        OneBlockIslands oneBlockIsland = getOneBlocksIsland(island);
+                        String hololine = oneBlockIsland.getHologram();
+                        if (hololine != null) {
+                            final Hologram hologram = HologramsAPI.createHologram(BentoBox.getInstance(), island.getCenter().add(0.5, 2.6, 0.5));
+                            for (String line : hololine.split("\\n")) {
+                                hologram.appendTextLine(ChatColor.translateAlternateColorCodes('&', line));
+                            }
+                        }
+                    }
+                }
+            }
+        }, 400L); // Wait 20 Seconds so that all Islands are Loaded
```

Comment:
This is a bit of a hack. You can override the Addon#allLoaded method instead. It gets called when BentoBox is fully loaded and all addons loaded.